pom.xml

```diff
@@ -215,25 +215,74 @@
       <artifactId>script-security</artifactId>
       <version>1.54</version>
     </dependency>
-    <!-- not sure if this makes any difference but attempting to fix enforcer errors-->
-    <!-- TODO rather use dependencyManagement if necessary -->
-    <dependency>
-      <groupId>joda-time</groupId>
-      <artifactId>joda-time</artifactId>
-      <version>2.10.1</version>
-    </dependency>
-    <dependency>
-      <groupId>io.fabric8</groupId>
-      <artifactId>kubernetes-client</artifactId>
-      <version>4.1.3</version>
-    </dependency>
-    <dependency>
-      <groupId>com.fasterxml.jackson.core</groupId>
-      <artifactId>jackson-annotations</artifactId>
-      <version>2.9.0</version>
-    </dependency>
   </dependencies>
 
+  <!--
+    This is to just remove the UpperBounds rule
+    This rule is covered by the tests that actually start jenkins and look for plugin start failures.
+    This needs to be kept in sync with the parent.
+  -->
+  <build>
+    <plugins>
+       <plugin>
+        <artifactId>maven-enforcer-plugin</artifactId>
+        <executions>
+          <execution>
+            <id>display-info</id>
+            <configuration>
+              <rules combine.self="override">
```

Comment:
the `combine.self` is the important part here saying "I do not care what the parent has - what I define is what I want" (the default is to merge).